docs/0.4/concepts/security_considerations.md

```diff
@@ -0,0 +1,164 @@
+# Security Considerations
+
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+Splinter provides several levels of security and customizable permissions.
+
+## Cryptographic signing with public/private keys
+
+Like Sawtooth and other distributed ledgers, Splinter uses cryptographic
```

Comment:
We should use "Hyperledger Sawtooth" and maybe make it a link to https://sawtooth.hyperleder.org/.  "Splinter uses" -- this is really a Scabbard detail so this should read something like "Scabbard, the default distributed ledger service that is built into Splinter, uses..."

Reply:
I rewrote this section to clarify the scabbard info and to add that the admin service uses keys for circuit management.